### PR TITLE
Alternate PR Comment

### PR DESCRIPTION
**Pull request by wanyaland**
- Create test cases for adding comment badge and comment body
- Fixed cla utility functions for appending comment/badge

Signed-off-by: Harold <wanyaland@gmail.com>

### CODE REVIEW

File: cla-backend/cla/utils.py [hunk 1]
```diff
@@ -51,11 +67,11 @@ def create_database(conf=None):
     """
     if conf is None:
         conf = cla.conf
-    cla.log.info('Creating CLA database in %s', conf['DATABASE'])
-    if conf['DATABASE'] == 'DynamoDB':
+    cla.log.info("Creating CLA database in %s", conf["DATABASE"])
+    if conf["DATABASE"] == "DynamoDB":
```
Comment (dealako) [inline cla-backend/cla/utils.py]: As far as I know, we use single quotes throughout the codebase. Can you revert the quote changes?

**Comment by dealako** [comment]
Can you revert the utils.py changes that simply change the single quote to a double quote? I'm unable to figure out what changes you presented due to the sheer number of diffs dues to the quote change.  Thanks.

**Comment by wanyaland** [comment]
> Can you revert the utils.py changes that simply change the single quote to a double quote? I'm unable to figure out what changes you presented due to the sheer number of diffs dues to the quote change. Thanks.

> Noted and reverted utils.py

**Comment by wanyaland** [comment]
Case Unsigned
Screenshot 2020-01-14 at 18 55 27

Case User Id is None
Screenshot 2020-01-14 at 18 24 09

**Pull request merged**